[2020-06-10 16:33:22.447,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 16:42:10.368,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 16:43:27.254,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 16:54:06.284,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 16:54:42.407,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 17:03:10.400,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 17:05:05.212,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 17:05:44.899,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 17:11:57.223,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 18:53:17.194,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 18:54:30.011,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 18:57:20.608,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 18:58:58.983,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 18:59:22.122,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 19:00:19.613,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-10 19:02:48.232,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-12 12:49:53.861,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:02:46.980,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:07:02.049,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:20:12.328,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:23:06.319,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:24:26.905,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:29:48.945,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:30:41.540,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:37:54.356,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:42:01.624,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:43:20.251,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:44:25.542,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 13:47:06.802,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 15:37:21.372,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 15:40:23.734,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 15:41:31.764,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 15:42:04.514,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:10:21.427,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:10:58.960,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:20:57.975,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:23:03.460,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:27:43.042,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:30:59.061,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:31:30.938,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 16:43:19.633,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 17:02:10.310,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:12:27.437,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:13:27.421,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:14:01.919,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:14:40.229,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:28:07.418,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:30:40.343,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:57:04.302,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:58:14.198,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 18:58:47.168,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 19:00:24.734,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 20:54:51.083,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 20:55:44.621,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 20:57:10.837,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-12 20:57:48.019,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 10:04:33.490,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 11:04:45.838,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 12:00:12.779,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 23:15:54.788,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 23:16:35.670,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 23:17:30.213,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 23:23:56.512,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-13 23:52:44.955,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:22.053,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:22.172,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:22.776,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:23.766,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:24.251,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:25.076,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:25.985,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:26.362,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:26.632,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:26.931,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:27.750,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:28.112,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:28.609,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:29.251,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:29.734,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:29.826,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:29.858,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:30.483,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:31.277,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:31.443,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:31.989,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:32.100,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:33.090,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:33.752,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:34.189,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:35.016,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:35.640,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:36.527,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:36.774,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:36.957,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:00:37.264,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:01:39.331,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:08:36.060,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 00:13:26.495,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 17:04:20.494,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-14 17:29:33.360,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-14 19:00:00.069,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-14 19:00:18.540,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-14 19:00:55.241,Pyro4.core,ERROR] cannot connect to ('192.168.0.119', 32000): [Errno 61] Connection refused
[2020-06-14 19:13:16.728,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:14:26.118,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:19:35.688,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:20:03.787,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:20:56.542,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:21:28.209,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:27:20.383,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:29:06.168,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:30:09.780,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:31:04.025,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:32:32.098,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:33:26.058,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:34:17.054,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:35:28.114,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 19:36:23.619,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 20:12:59.305,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
[2020-06-14 20:13:28.515,Pyro4.core,ERROR] cannot connect to ('127.0.0.1', 32000): [Errno 61] Connection refused
